spec/oh-my-fish.spec.fish

```
@@ -1,6 +1,6 @@
-import plugins/fish-spec
+import plugins/fish-specs
```

So it was silently failing to load the plugin, but since it was already loaded in the session it didn't complain. Ha.